.goreleaser.yml

```diff
@@ -16,11 +16,18 @@ builds:
   ldflags:
     - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
   goos:
+    - freebsd
+    - windows
```

Comment:
![5351aa7e85011caa163f670450d6a08](https://user-images.githubusercontent.com/15064560/223496097-10687093-2fc3-4ec2-bb1d-27d61e084510.png)
![e312a1b9b1e7644a3cfebf9dc87c79d](https://user-images.githubusercontent.com/15064560/223496147-f0c13c58-8e14-48e1-84e3-bf6864d652de.png)
It does not compile successfully on freebsd and windows